Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
+        --build.cmd "go build -o tmp/bin/main" --build.bin "tmp/bin/main" --build.delay "100" \
```

Comment:
Instead of using a hard-coded version (`@v1.61.1`) in the `dev_server` task, consider using a variable so that it can be more easily managed and updated if needed in the future.